src/app/src/components/Submissions/Detail/Detail.js

```diff
@@ -53,19 +46,22 @@ export default function SubmissionDetail() {
         >
             <Flex mb={7}>
                 <Flex direction='column' w='50%'>
-                    <Flex>
+                    <Flex alignItems='center'>
                         <IconButton
                             icon={<Icon as={ArrowLeftIcon} />}
                             aria-label='Back'
                             mr={6}
                             onClick={() => navigate('/submissions')}
                         />
                         <Heading size='lg' mr={6}>
-                            {submissionName}
+                            {details.name}
                         </Heading>
-                        <SubmittedBadge variant='submissionDetail' />
+                        <StatusBadge status={details.status} fixedHeight />
                     </Flex>
-                    <Info submission={submission} />
+                    <Info
+                        primaryContact={details.submission.primary_contact}
```

Comment:
Since we're using snake_case everywhere else, should do so here too

```suggestion
                        primary_contact={details.submission.primary_contact}
```